README.md

```diff
@@ -119,15 +119,15 @@ Beta features:
 - Transport client strategies
   - Proxyless strategies
     - [ ] Encrypted DNS
-    - [x] Packet splitting
+    - [x] Packet splitting ([reference](https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/transport/split))
```

Comment:
*nit* we can actually put the link around "Packet splitting" itself.

Reply:
I put it on the side to be consistent with the other entries. It allows us to have multiple links for a feature, as for "Library to run a local SOCKS5 or HTTP-Connect proxy"